packages/core/stories/toolkit-provider.doc.stories.mdx

```diff
@@ -203,30 +122,101 @@ function MyApp() {
 
 <Canvas>
-  <Story name="Portal">
-    <ToolkitProvider density="touch">
-      <Button variant="cta">CTA</Button>
-      <Button variant="primary">Primary</Button>
-      <Portal>
-        <Button variant="secondary">Secondary</Button>
-      </Portal>
+  <Story name="Simple ToolkitProvider - Touch Density">
+    <ToolkitProvider density="high" mode="light">
+      <Card>
+        <div>
+          <h1>This is Card</h1>
+          <span>Using Nested DOM Elements</span>
+        </div>
+      </Card>
     </ToolkitProvider>
   </Story>
-</Canvas> -->
+</Canvas>
+
+Here is another example of a `Card` containing an additional nested `Card`. The nested `Card` is also wrapped with a ToolkitProvider, this time configured to use dark mode. Density is not specified on the inner ToolkitProvider, so is inherited from the outer instance.
+
+<Canvas>
+  <Story name="Nested ToolkitProvider">
+    <NestedProviders />
+  </Story>
+</Canvas>
+
+By default, the `ToolkitProvider` wraps nested content with a `div` element to which the theme and density CSS classes are added.
+Given the following JSX:
+
+```jsx
+<ToolkitProvider mode="light" density="medium">
+  <Card>...</Card>
+</ToolkitProvider>
+```
+
+then we would see the following HTML:
+
+```html
+<div class="utik-theme uitk-density-medium" data-mode="light">
+  <div class="uitkCard">...</div>
+</div>
+```
+
+The `div` element is styled with `display: contents` using the class `uitk-theme` which generally ensures that it does not break any layout or formatting style rules that may be in effect. Child elements are styled and laid out as though the containing `div` element were not present.
+
+Some browser implementations of `display: contents` are buggy (Safari, for example). Further, your code may rely on CSS query selectors in JavaScript that might be broken if they do not anticipate the additional `div` element inserted by the ToolkitProvider.
+
+If you are concerned about either of these scenarios, the `ToolkitProvider` can be configured to not inject this additional element.
+Instead it can add the required CSS classes to the child element nested within the Provider. You may also simply prefer this approach.
+
+There is (currently) one limitation to this pattern, you must only use a single child element. This is enforced within the code and, for TypeScript users, via the typings. This would look as follows:
+
+```jsx
+<ToolkitProvider mode="light" density="medium" applyClassesTo="child">
+  <Card>...</Card>
+</ToolkitProvider>
+```
+
+Then we would see the following HTML.
+
+```html
+<div class="uitkCard uitk-theme uitk-density-medium" data-mode="light">...</div>
+```
+
+The `ToolkitProvider` can also be configured to add the required CSS classes to the root HTML element. This would allow you to apply styles to the whole of the document rather than just the root `div` that React renders to. Some use cases for this would be to prevent overscroll or to apply styling to the whole background of the page.
```

Comment:
This terminology is a bit confusing: "rather than just the root `div` that React renders to"

Reply:
How about "rather than just the `div` given to ReactDOM to render to"

Reply:
I wonder if we could just get rid of the sentence. It doesn't add much

```suggestion
The `ToolkitProvider` can also be configured to add the required CSS classes to the root HTML element. This would allow you to apply styles to the whole of the document. Some use cases for this would be to prevent overscroll or to apply styling to the whole background of the page.
```